packages/block-library/src/file/edit.js

```diff
@@ -205,43 +207,47 @@ class FileEdit extends Component {
 						</Toolbar>
 					</MediaUploadCheck>
 				</BlockControls>
-				<div className={ classes }>
-					<div className={ 'wp-block-file__content-wrapper' }>
-						<RichText
-							wrapperClassName={ 'wp-block-file__textlink' }
-							tagName="div" // must be block-level or else cursor disappears
-							value={ fileName }
-							placeholder={ __( 'Write file name…' ) }
-							withoutInteractiveFormatting
-							onChange={ ( text ) => setAttributes( { fileName: text } ) }
-						/>
-						{ showDownloadButton &&
-							<div className={ 'wp-block-file__button-richtext-wrapper' }>
-								{ /* Using RichText here instead of PlainText so that it can be styled like a button */ }
+				<Animate type={ animate && isBlobURL( href ) ? 'loading' : null }>
```

Comment:
```suggestion
				<Animate type={ isBlobURL( href ) ? 'loading' : null }>
```

This should be enough here